Bingha/Bingha/Views/Stat/StatViewController.swift

```diff
@@ -10,49 +10,142 @@ import UIKit
     }
     
-    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, referenceSizeForHeaderInSection section: Int) -> CGSize {
-        if section == 1 || section == 2 {
-            return CGSize(width: collectionView.bounds.width, height: 50)
+    //컬렉션뷰 섹션별로 셀 가져와서 업데이트
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        switch indexPath.section {
+            
+        // 세그먼트 컨트롤. 태그번호로 가져오고 첫번째 세그먼트 기본으로.
+        case 0:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: segmentID, for: indexPath)
+            if let segment = cell.viewWithTag(0) as? UISegmentedControl {
+                segment.selectedSegmentIndex = 0
+                //TODO: 세그먼트 컨트롤 동작 구현하기
+            }
+            return cell
+            
+        //탄소저감량 섹션
+        case 1:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: reducedID, for: indexPath)
+            return cell
+        
+        //비교 헤더 섹션
+        case 2:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: compareHeaderID, for: indexPath)
+            return cell
+            
+        //비교 섹션
+        case 3:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: compareID, for: indexPath) as! CompareCollectionViewCell
+            let compareInfo = compareViewModel.CompareModelInfo(at: indexPath.item)
+            cell.update(info: compareInfo)
+            return cell
+            
+        //걷기 모음 헤더 섹션
+        case 4:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: statisticsHeaderID, for: indexPath)
+            return cell
+            
+        //걷기 모음 섹션
+        case 5:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: statisticsID, for: indexPath) as! StatisticsCollectionViewCell
+            let statisticInfo = statisticsViewModel.statisticsInfo(at: indexPath.item)
+            cell.update(info: statisticInfo)
+            return cell
+            
+        default:
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: statisticsID, for: indexPath) as! StatisticsCollectionViewCell
+            let statisticInfo = statisticsViewModel.statisticsInfo(at: indexPath.item)
+            cell.update(info: statisticInfo)
+            return cell
         }
-        return CGSize()
     }
     
+    //컬렉션뷰 섹션 안 셀 크기 설정
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        let itemSpacing: CGFloat = 10
-        let width: CGFloat = (collectionView.bounds.width - itemSpacing) / 2.2
-        return CGSize(width: width, height: width)
+        switch indexPath.section {
+        case 0:
+            let width: CGFloat = collectionView.bounds.width-40
```

Comment:
반복되는 width와 날짜별 cell의 width를 분리해서 상수로 빼놓으면 사용하기 편리할 것 같습니다!
```suggestion
            let defaultWidth: CGFloat = collectionView.bounds.width-40
            let (각 날짜별 셀이름)Width: CGFloat = (collectionView.bounds.width - itemSpacing) / 2.25
```